src/zeit/nightwatch/pytest.py

```diff
@@ -35,11 +35,11 @@ def selenium_session(request, nightwatch_config):
     config.setdefault('driver_class', 'WebDriverChrome')
     cls = getattr(zeit.nightwatch, config.pop('driver_class'))
     browser = cls(**config)
-    request.addfinalizer(browser.quit)
-    return browser
+    yield browser
+    browser.quit
```

Comment:
Die Funktion muss man in dieser Formulierung dann ganz normal `aufrufen()`, damit etwas passiert.

Reply:
Hab ich gemacht. Danke für's Anschauen.